docs-src/guide-rpc-server-api.scrbl

```diff
@@ -0,0 +1,333 @@
+
+@(define link-deploy-rpc @reachexlink[
+  #:ref "1474d32"
+  #:loc "97"
+  "tut-6-rpc/client-py/index.py"
+  "RPC code from the \"tut-6-rpc\""])
+
+Contrast @link-deploy-stdlib example:
+@js{
+  console.log(`Alice will deploy the contract.`);
+  const ctcAlice = alice.deploy(backend);
+}
+
+...with the equivalent @link-deploy-rpc Python example:
+@py{
+  ctc_alice = rpc('/acc/deploy', acc_alice)
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "acc-attach"]{/acc/attach}
+
+The @urlin{/acc/attach} route corresponds to an @jsin{attach} operation on
+@tech{accounts} when using the standard library directly.
+
+@(define link-attach-stdlib @reachexlink[
+  #:ref "27fb2a9"
+  #:loc (cons "14" "15")
+  "maybe-send/index.mjs"
+  "this code from the \"maybe-send\""])
+
+@(define link-attach-rpc @reachexlink[
+  #:ref "1474d32"
+  #:loc "98"
+  "tut-6-rpc/client-py/index.py"
+  "RPC code from the \"tut-6-rpc\""])
+
+Contrast @link-attach-stdlib example:
+@js{
+  console.log(`Bob will attach to the contract.`);
+  const ctcBob = bob.attach(backend, ctcAlice.getInfo());
+}
+
+...with the equivalent @link-attach-rpc Python example:
+@py{
+  ctc_bob = rpc('/acc/attach', acc_bob, rpc('/ctc/getInfo', ctc_alice))
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "ctc-getinfo"]{/ctc/getInfo}
+
+The @urlin{/ctc/getInfo} route corresponds to a @jsin{getInfo} operation on
+@tech{contracts} when using the standard library directly.
+
+Contrast @link-attach-stdlib example:
+@js{
+  console.log(`Bob will attach to the contract.`);
+  const ctcBob = bob.attach(backend, ctcAlice.getInfo());
+                                     ------------------
+}
+
+...with the equivalent @link-attach-rpc Python example:
+@py{
+  ctc_bob = rpc('/acc/attach', acc_bob, rpc('/ctc/getInfo', ctc_alice))
+                                        ------------------------------
+}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "forget-acc"]{/forget/acc}
+See: @secref{id-clean-up}.
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "forget-ctc"]{/forget/ctc}
+See: @secref{id-clean-up}.
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@subsection[#:tag "backend"]{/backend/*}
+
+@(define link-backend-stdlib @reachexlink[
+  #:ref "27fb2a9"
+  #:loc (cons "41" "52")
+  "tut-6/index.mjs"
+  "this code from the (non-RPC) \"tut-6\""])
+
+@(define link-backend-rpc @reachexlink[
+  #:ref "1474d32"
+  #:loc (cons "122" "144")
+  "tut-6-rpc/client-py/index.py"
+  "RPC code from the \"tut-6-rpc\""])
+
+The @urlin{/backend} namespace is dynamically populated with routes which are
+specific to your @|DApp|, e.g. @urlin{/backend/Alice}, and
+@urlin{/backend/Bob}, or @urlin{/backend/SomeOtherParticipant} whose logic
+you've encoded.
```

Comment:
You could say "The /backend/ route corresponds to the backend that Reach compiles. Any participant backends it contains will be exposed as routes. For example, if your Reach program defines participants 'Alice' and 'Bob', then it will expose a `/backend/Alice` and `/backend/Bob` route."